Resources/views/PageAdmin/tree.html.twig

```diff
@@ -48,8 +48,8 @@ file that was distributed with this source code.
                 <h1 class="box-title">
                     {{ 'pages.tree_site_label'|trans({}, 'SonataPageBundle') }}
                     <div class="btn-group">
-                        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-                            <strong class="text-info">{{ currentSite.name }}</strong> <span class="caret"></span>
+                        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" {{ currentSite is null ? 'disabled' }}>
```

Comment:
Per the [html5 spec](https://www.w3.org/TR/html5/forms.html#enabling-and-disabling-form-controls:-the-disabled-attribute) it is valid. Even per the [html4 spec](https://www.w3.org/TR/html4/interact/forms.html#adef-disabled). 

Only in [xhtml spec](https://www.w3.org/TR/xhtml1/dtds.html#dtdentry_xhtml1-strict.dtd_button) is `disabled="disabled"` required.